kv/dist_sender.go

```diff
@@ -480,6 +472,131 @@ func (ds *DistSender) sendRPC(raftID int64, replicas replicaSlice,
 	return err
 }
 
+// sendAttempt is invoked by Send and handles retry logic and cache eviction
+// for a call sent to a single range.
```

Comment:
Why not mention the return values in comment? In particular, the retry.Status.

    // This method returns a retry status, which is Break on success and either
    // Break, Continue or Reset depending on error condition. This method is
    // expected to be invoked from within a backoff / retry loop to retry the send
    // repeatedly (e.g. to continue processing after a critical node becomes
    // available after downtime or the range descriptor is refreshed via lookup).